CMakeLists.txt

```diff
@@ -296,7 +296,7 @@ if(HIP_PLATFORM STREQUAL "hcc")
     endif()
     add_library(hip_hcc SHARED ${SOURCE_FILES_RUNTIME})
     add_library(hip_hcc_static STATIC ${SOURCE_FILES_RUNTIME})
-    if(HIP_COMPILER STREQUAL "hcc")
+    if(HIP_COMPILER STREQUAL "hcc" OR HIP_COMPILER STREQUAL "clang")
```

Comment:
Why are we adding HIP-Clang here? I didn't add clang here because it shouldn't have hc_am, and also it doesn't support hiprtc. We have our own implementation in VDI.

Reply:
CMakeLists.txt is currently being used only for building HIP-HCC. To enabling testing of HIP-HCC with HIP-clang compiler, enabled generation of hiprtc for clang path.